supervisor/api/network.py

```diff
@@ -111,7 +111,7 @@ def interface_struct(interface: Interface) -> dict:
         ATTR_IPV4: ipconfig_struct(interface.ipv4) if interface.ipv4 else None,
         ATTR_IPV6: ipconfig_struct(interface.ipv6) if interface.ipv6 else None,
         ATTR_WIFI: wifi_struct(interface.wifi) if interface.wifi else None,
-        ATTR_VLAN: wifi_struct(interface.vlan) if interface.vlan else None,
+        ATTR_VLAN: interface.vlan.id if interface.vlan else None,
```

Comment:
Need to be: `{"id": interface.vlan.id, "parent": interface.vlan.parent}`

Reply:
https://developers.home-assistant.io/docs/api/supervisor/models#vlan-configuration

Reply:
`parent` does not exist on `VlanConfig`

```
@attr.s(slots=True)
class VlanConfig:
    """Represent a vlan configuration."""

    id: int = attr.ib()
    interface: str = attr.ib()

```

Reply:
`{"id": interface.vlan.id, "parent": interface.vlan.interface}`